pkg/store/prometheus.go

```diff
@@ -607,3 +607,25 @@ func (p *PrometheusStore) LabelValues(ctx context.Context, r *storepb.LabelValue
 	sort.Strings(vals)
 	return &storepb.LabelValuesResponse{Values: vals}, nil
 }
+
+func (p *PrometheusStore) LabelSet() []labelpb.ZLabelSet {
+	lset := p.externalLabelsFn()
+
+	labels := make([]labelpb.ZLabel, 0, len(lset))
+	labels = append(labels, labelpb.ZLabelsFromPromLabels(lset)...)
+
+	// Until we deprecate the single labels in the reply, we just duplicate
```

Comment:
I don't think this comment makes sense anymore? :thinking: We should remove this